lightly_studio/src/lightly_studio/db_manager.py

```diff
@@ -41,7 +41,14 @@ def __init__(
         self._engine_url = engine_url if engine_url else "duckdb:///lightly_studio.db"
         if cleanup_existing:
             _cleanup_database_file(engine_url=self._engine_url)
-        self._engine = create_engine(url=self._engine_url, poolclass=poolclass)
+        # Total available connections = pool_size + max_overflow = 20 + 40 = 60
+        # (default is (pool_size=5, max_overflow=10))
+        self._engine = create_engine(
+            url=self._engine_url,
+            poolclass=poolclass,
+            pool_size=20,
```

Comment:
Let's do pool size of 10. (default of duckdb is 10. And 5 was default in SQLAlchemy)